packages/fluentui/react-northstar/src/components/TextArea/TextArea.tsx

```diff
@@ -106,21 +115,29 @@ export const TextArea: ComponentWithAs<'textarea', TextAreaProps> &
 
   const handleChange = (e: React.ChangeEvent | React.FormEvent) => {
     const newValue = _.get(e, 'target.value');
-
     _.invoke(props, 'onChange', e, { ...props, value: newValue });
     setValue(newValue);
   };
 
+  React.useLayoutEffect(() => {
+    if (autoResize) {
+      inputRef.current.style.height = '0';
+      inputRef.current.style.height = `${inputRef.current?.scrollHeight}px`;
+    }
+  }, [inputRef.current?.scrollHeight, autoResize]);
```

Comment:
During the first render `inputRef.current.scrollHeight` will be null. In general it's incorrect to use such things as effect dependencies